Src/Model.cpp

```diff
@@ -49,15 +52,33 @@ ModelMeshPart::~ModelMeshPart()
 _Use_decl_annotations_
 void ModelMeshPart::Draw(_In_ ID3D12GraphicsCommandList* commandList) const
 {
+    if (!indexBufferSize || !vertexBufferSize)
```

Comment:
This validation will ensure that developers moving to the new version won't get silent failures...